docs/legacy/getting-started-apm-server.asciidoc

```diff
@@ -330,3 +445,61 @@ include::{libbeat-dir}/repositories.asciidoc[]
 
 // Shared docker
 include::{libbeat-dir}/shared-docker.asciidoc[]
+
+
+=== Fleet-managed APM Server
+
+This guide will explain how to set up and configure a Fleet-managed APM Server.
+
+[float]
+==== Prerequisites
+
+You need {es} for storing and searching your data, and {kib} for visualizing and managing it.
+When setting these components up, you need:
+
+include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/prereq.asciidoc[tag=self-managed]
+
+==== Step 1: Set up Fleet
+
+Use {fleet} in {kib} to get APM data into the {stack}.
+The first time you use {fleet}, you'll need to set it up and add a
+{fleet-server}:
+
+include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc[tag=self-managed]
+
+For more information, refer to {fleet-guide}/fleet-server.html[{fleet-server}].
+
+==== Step 2: Add and configure the APM integration
+
+include::{obs-repo-dir}/observability/tab-widgets/add-apm-integration/content.asciidoc[tag=self-managed]
+
+==== Step 3: Install APM agents
+
+APM agents are written in the same language as your service.
+To monitor a new service, you must install the agent and configure it with a service name,
+APM Server host, and Secret token.
+
+* **Service name**: The APM integration maps an instrumented service's name–defined in each {apm-agent}'s configuration–
+to the index that its data is stored in {es}.
+Service names are case-insensitive and must be unique.
+For example, you cannot have a service named `Foo` and another named `foo`.
+Special characters will be removed from service names and replaced with underscores (`_`).
+
+* **APM Server URL**: The host and port that APM Server listens for events on.
+This should match the host and port defined when setting up the APM integration.
+
+* **Secret token**: Authentication method for {apm-agent} and APM Server communication.
+This should match the secret token defined when setting up the APM integration.
```

Comment:
Step 2 mentions to write down the apm-server url as it is needed afterwards, no mentioning of the secret-token. But here the secret token is required. Either we describe to enable api-key auth and make it part of this instructions, or we add the `secret-token` in Step 2.